rank/scripts/createIndex.ts

```diff
@@ -78,20 +72,17 @@ async function go() {
       body: {
         source: {
           index: localIndex,
+          size: 100, // batch size reduced to avoid memory issues
```

Comment:
reduced from what?

Reply:
longer comment added in https://github.com/wellcomecollection/catalogue-api/pull/310/commits/e6abd8e59aaa92d686c2afa1783fe4b6faebbe4d !